dev/run-tests.py

```diff
@@ -342,7 +342,7 @@ def build_spark_unidoc_sbt(extra_profiles):
     set_title_and_block("Building Unidoc API Documentation", "BLOCK_DOCUMENTATION")
     # Enable all of the profiles for the build:
     build_profiles = extra_profiles + modules.root.build_profile_flags
-    sbt_goals = ["unidoc"]
+    sbt_goals = ["clean", "unidoc"]
```

Comment:
Hm.. I avoided this because it will slow down doc gen since it re-uses complied ones from the regular build. How long does it increase?

Reply:
Also, if I am not wrong, this wont verify the documentation in test codes anymore. That's fine as documentation in the test codes wont be the part of our official documentation though. Cc @JoshRosen I think I talked about this with you before somewhere.